src/data/store.test.js

```diff
@@ -60,17 +60,17 @@ describe('store aggregator module', () => {
         });
       });
       describe('if no SEGMENT_KEY', () => {
-        const key = configuration.SEGMENT_KEY;
+        const key = getConfig().SEGMENT_KEY;
         beforeEach(() => {
-          configuration.SEGMENT_KEY = false;
+          getConfig.mockImplementation(() => ({ SEGMENT_KEY: false }));
```

Comment:
If this is still failing, you could try doing [what frontend-app-learning does](https://github.com/openedx/frontend-app-learning/blob/master/src/instructor-toolbar/InstructorToolbar.test.jsx#L10-L15).

Alternatively, [what EdxHeader does](https://github.com/openedx/frontend-app-gradebook/blob/master/src/components/EdxHeader/test.jsx#L18) (ignoring for a minute the question of what is edX-specific code still doing here), in this repo.

Reply:
@arbrandes I find myself lacking understanding of Jest's mock system, particularly around when patches are enabled and when they are reset. In my head, I'm trying to equate it to Python's mocking framework but that's only getting me so far.

In interest of time, I feel like this is a minor wart that we can leave as-is for now.  Do you agree?

Reply:
Agreed!  Plus, I didn't realize you had pulled in a fix for the failing test when I wrote that comment.